### DISCUSSION TITLE
QR Reader Plugin that works with Cordova-Android 11?

### DISCUSSION THREAD

**Discussion by phusting** [apache/cordova#360] [Q&A]
Any good suggestions for a QR Reader Plugin that works with Cordova-Android 11?  I've been using cordova-plugin-qr-barcode-scanner 8.0.3 in the past, but it doesn't work with the latest Android platform.

I would love to get a good suggestion

Thank you

**Comment by phusting** [comment]
Anyone using this one?  https://www.npmjs.com/package/cordova-plugin-qrscanner-11   Any comments?

**Comment by iamrakesh** [comment]
We forked github repo and made minor changes to get it working with cordova-android@11
https://github.com/iamrakesh/phonegap-plugin-barcodescanner/commit/0852232a028f72a3c81d446f75cb93786917c070

**Reply by phusting** [reply]
What does the commandline look like to add this to my project.  I'm a bit of a newbe.  :)

**Reply by phusting** [reply]
I tried cordova plugin add @iamrakesh/phonegap-plugin-barcodescanner@0852232 and cordova plugin add @iamrakesh/phonegap-plugin-barcodescanner with no success

**Reply by breautek** [reply]
Assuming the package is not published, you can install it via git. You'll need to have git installed (and it be available in your `PATH` variable).

```
cordova plugin add https://github.com/iamrakesh/phonegap-plugin-barcodescanner.git#0852232a028f72a3c81d446f75cb93786917c070
```

`https://github.com/iamrakesh/phonegap-plugin-barcodescanner.git` is the git https url for cloning. `#0852232a028f72a3c81d446f75cb93786917c070` is the commit hash so that it clones specifically at https://github.com/iamrakesh/phonegap-plugin-barcodescanner/commit/0852232a028f72a3c81d446f75cb93786917c070
